src/View/Helper/FormHelper.php

```diff
@@ -41,15 +41,16 @@ class FormHelper extends Helper
      */
     protected $_templates = [
         'dateWidget' => '<ul class="list-inline"><li class="year">{{year}}</li><li class="month">{{month}}</li><li class="day">{{day}}</li><li class="hour">{{hour}}</li><li class="minute">{{minute}}</li><li class="second">{{second}}</li><li class="meridian">{{meridian}}</li></ul>',
-        'error' => '<div class="help-block">{{content}}</div>',
-        'help' => '<div class="help-block">{{content}}</div>',
+        'error' => '<div class="invalid-feedback">{{content}}</div>',
+        'help' => '<small id="{{name}}" class="form-text text-muted">{{content}}</small>',
```

Comment:
Is adding `id` attribute necessary? I don't see any change in code to set the `name` placeholder.